5.10.0/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:7
+FROM java:7-jre
```

Comment:
I would even prefer if we stay on java:7-jre. We can change this later to jre if we want. I am fine with java8-jre for the new images